e2e/README.md

```diff
@@ -42,187 +33,99 @@ Tests can be run using a Makefile target under the e2e directory. `e2e/Makefile`
+### Example of running a full testsuite
 
-The relayer should be started as part of the test if required. See [Starting the Relayer](#starting-the-relayer)
+> NOTE: not all tests may support full parallel runs due to possible chain wide modifications such as params / gov
+> proposals / chain restarts.
 
-```go
-relayer, channelA := s.SetupChainsRelayerAndChannel(ctx, s.FeeMiddlewareChannelOptions())
-chainA, chainB := s.GetChains()
+```sh 
+make e2e-suite entrypoint=TestTransferTestSuite
 ```
 
-### Creating test users
-
-There are helper functions to easily create users on both chains.
-
-```go
-chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
-chainBWallet := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
-```
-
-### Waiting
-
-We can wait for some number of blocks on the specified chains if required.
-
-```go
-chainA, chainB := s.GetChains()
-err := test.WaitForBlocks(ctx, 1, chainA, chainB)
-s.Require().NoError(err)
-```
+Similar to running a single test, if `jq` and `fzf` are installed you can run `make e2e-suite` and be prompted
+to interactively select a test suite to run.
 
-### Query wallet balances
+### Running tests outside the context of the Makefile
 
-We can fetch balances of wallets on specific chains.
-
-```go
-chainABalance, err := s.GetChainANativeBalance(ctx, chainAWallet)
-s.Require().NoError(err)
-```
-
-### Broadcasting messages
-
-We can broadcast arbitrary messages which are signed on behalf of users created in the test.
-
-This example shows a multi message transaction being broadcast on chainA and signed on behalf of chainAWallet.
-
-```go
-relayer, channelA := s.SetupChainsRelayerAndChannel(ctx, s.FeeMiddlewareChannelOptions())
-chainA, chainB := s.GetChains()
-
-chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
-chainBWallet := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
-
-t.Run("broadcast multi message transaction", func(t *testing.T){
-  msgPayPacketFee := feetypes.NewMsgPayPacketFee(testFee, channelA.PortID, channelA.ChannelID, chainAWallet.Bech32Address(chainA.Config().Bech32Prefix), nil)
-  msgTransfer := transfertypes.NewMsgTransfer(channelA.PortID, channelA.ChannelID, transferAmount, chainAWallet.Bech32Address(chainA.Config().Bech32Prefix), chainBWallet.Bech32Address(chainB.Config().Bech32Prefix), clienttypes.NewHeight(1, 1000), 0)
-  resp, err := s.BroadcastMessages(ctx, chainA, chainAWallet, msgPayPacketFee, msgTransfer)
-
-  s.AssertValidTxResponse(resp)
-  s.Require().NoError(err)
-})
-```
-
-### Starting the relayer
-
-The relayer can be started with the following.
-
-```go
-t.Run("start relayer", func(t *testing.T) {
-  s.StartRelayer(relayer)
-})
-```
+In order to run tests outside the context of the Makefile (e.g. from an IDE)
 
-### Arbitrary commands
+The main thing that needs to be set is the `E2E_CONFIG_PATH` environment variable. This should point to a valid config
```

Comment:
This is not entirely true. You don't have to set it, the default location works (this is what I currently use).

Reply:
ah good point, will just make a note about this env being what you need to change.